e2e/test/netstandard20/ProvisioningCertificateValidationTest.cs

```diff
@@ -29,30 +31,29 @@ public ProvisioningCertificateValidationTest()
         public async Task ProvisioningServiceClient_QueryInvalidServiceCertificateHttp_Fails()
         {
             using (var provisioningServiceClient = 
-                ProvisioningServiceClient.CreateFromConnectionString(
+                ProvisioningServiceClientFactory.CreateFromConnectionString(
                     Configuration.Provisioning.ConnectionStringInvalidServiceCertificate))
             {
-                Query q = provisioningServiceClient.CreateEnrollmentGroupQuery(
-                    new QuerySpecification("SELECT * FROM enrollmentGroups"));
+                var querySpecification = new QuerySpecification("SELECT * FROM enrollmentGroups");
 
-                var exception = await Assert.ThrowsExceptionAsync<ProvisioningServiceClientTransportException>(
-                    () => q.NextAsync()).ConfigureAwait(false);
+                var exception = await Assert.ThrowsExceptionAsync<HttpRequestException>(
+                    () => provisioningServiceClient.QueryEnrollmentGroupsAsync(querySpecification)).ConfigureAwait(false);
 
 #if NET47 || NET451
-                Assert.IsInstanceOfType(exception.InnerException.InnerException.InnerException, typeof(AuthenticationException));
+                Assert.IsInstanceOfType(exception.InnerException.InnerException, typeof(AuthenticationException));
 #elif NETCOREAPP2_0
```

Comment:
We don't support netcoreapp 2.0 anymore - the framework is deprecated. Please remove all #if defs.